File: app/src/main/java/by/d1makrat/library_fm/model/Artist.java
package by.d1makrat.library_fm.model;

public class Artist{

    private String mName;
    private String mListenersCount;
    private String mUrl;
    private String mImageUri;

    public String getName() {
        return mName;
    }

    public void setName(String pName) {
        mName = pName;
    }

    public String getListenersCount() {
        return mListenersCount;
    }

    public void setListenersCount(String pListenersCount) {
        mListenersCount = pListenersCount;
    }

    public String getUrl() {
        return mUrl;
    }

    public void setUrl(String pUrl) {
        mUrl = pUrl;
    }

    public String getImageUri() {
        return mImageUri;
    }

    public void setImageUri(String pImageUri) {
        mImageUri = pImageUri;
    }
}

File: app/src/main/java/by/d1makrat/library_fm/model/Artist.kt
package by.d1makrat.library_fm.model

open class Artist(val name: String,
                  val listenersCount: Int? = null,
                  val playCount: Int? = null,
                  val url: String? = null,
                  val imageUrl: String? = null,
                  val rank: String? = null,
                  val stats: Stats? = null) {

    constructor(name: String, imageUrl: String?, playCount: Int, rank: String) : this(name, null, playCount, null, imageUrl, rank)

    constructor(name: String, listenersCount: Int, url: String, imageUrl: String?) : this(name, listenersCount, null, url, imageUrl, null)
}
